pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -88,6 +88,12 @@
       ::    the :binding into a (map (unit @t) (trie knot =action)).
       ::
       bindings=(list [=binding =duct =action])
+      ::  cache: mapping from path->entry
+      ::
+      cache=(axal (list (unit cache-entry)))
```

Comment:
The historical bindings (`(list (unit cache-entry))`) seem to be entirely unaccessible. Is the plan to expose them through %eyre's namespace?. If so, Is the position in the list intended to correspond to a `%ud` case?

Also, will it be possible to delete entries other than the most recent? I'm worried about a space leak here.

Reply:
The plan is that while we have no need to expose them through `%eyre`'s namespace, that they could be exposed in that manner, with the position in the list corresponding to the case (backwards, highest case is most recent in list). I think that previous bindings should be able to be deleted using the `+del-cache` method, perhaps with an optional case. That is not currently implemented, but it should be @lukechampine.

Reply:
ok. How are optional arguments usually implemented? `(unit @ud)`? Also, we aren't currently exposing the case anywhere (nor the full list of entries), which makes it pretty annoying for a caller to figure out which case to specify. Perhaps the case could be included in the `%cache-changed` gift? 🤔

Reply:
`(unit @ud)` should suffice. Including the case in the `%cache-changed` gift is a great idea. Down the line it's likely that the eyre scry namespace will allow querying some of these details as well.